packages/grids/Container.jsx

```diff
@@ -1,31 +1,65 @@
+/* eslint-disable security/detect-object-injection */
 import React from "react";
+import { cnb } from "cnbuilder";
 
 /**
  * Container package
  *
  * @param {string} children The child elements that sit inside the container
  * @param {string} width The width of the container
- * @param {string} paddingX Include padding for the container
+ * @param {string} paddingX Horizontal padding for the container
+ * @param {string} paddingY Vertical padding for the container
  * @returns {JSX.Element}
  * @constructor
  */
-export function Container({ children, width = "large", paddingX = true }) {
-  const hasChildren = children !== undefined;
 
-  // CSS for these is defined in global.css to allow for override on basic story
-  const widthClasses = new Map();
-  widthClasses.set("narrow", "su-container-narrow");
-  widthClasses.set("large", "su-container-large");
-  widthClasses.set("wide", "su-container-wide");
-  widthClasses.set("full", "su-container-full");
+// CSS for these is defined in global.css to allow for override on basic story
+const widthClasses = {
+  narrow: "su-container-narrow",
+  large: "su-container-large",
+  wide: "su-container-wide",
+  full: "su-container-full",
+  /**
+   * Centered container class from Decanter
+   * https://github.com/SU-SWS/decanter/blob/main/src/plugins/components/layout/centered-container.js
+   */
+  cc: "su-cc",
+};
```

Comment:
Did a small refactor using object - this is what we were doing for most Storyblok sites and it's a bit more performant and easier to read than using maps.

Reply:
Very nice. Love it! Thanks.